pyglidein/startd_cron_scripts/clsim_gpu_test.py

```diff
@@ -22,6 +21,15 @@ def main():
     (options, args) = parser.parse_args()
 
     try:
+        env = os.environ
+        gpus = env.get('GPUS', '0')
+        if gpus == '0':
+            print 'PYGLIDEIN_RESOURCE_GPU=False'
+            print '- update:true'
+            sys.exit(0)
```

Comment:
`sys.exit` is frowned upon.  would be better to restructure the code so this isn't necessary

Reply:
I forgot how much messing up I did to the script to add the crazy debugging logic.  I can clean this up a lot.